src/test/java/com/github/virtualgamers/hrtg/rest/controller/PartyControllerIT.java

```diff
@@ -0,0 +1,110 @@
+package com.github.virtualgamers.hrtg.rest.controller;
```

Comment:
I've never seen test files end with IT instead of Test, what does it stand for?

Reply:
Integration Test

Reply:
👍 thanks!